js/partials.js

```diff
@@ -1,3 +1,45 @@
+const RubySummit = {
+  renderSchedule: function() {
+    fetch("data/schedule.json")
+    .then(response => {
+      return response.json()
+    })
+    .then(schedule => {
+      schedule.map(track => {
```

Comment:
```suggestion
      schedule.forEach(track => {
```
Não temos nenhum return nessa função

Reply:
Opa @Roriz, como esse código não é novo, vou evitar fazer qualquer modificação nele. Seria legal ver com o pessoal de abrir um PR separado para mudar essa implementação.

Reply:
@Roriz pode abrir um PR com essas modificações?